src/Microsoft.AspNet.Authentication.OAuth/OAuthAuthenticationHandler.cs

```diff
@@ -159,14 +160,44 @@ protected override AuthenticationTicket AuthenticateCore()
             return new TokenResponse(oauth2Token);
         }
 
-        protected virtual async Task<AuthenticationTicket> GetUserInformationAsync(AuthenticationProperties properties, TokenResponse tokens)
+        protected virtual async Task<AuthenticationTicket> CreateTicketAsync(AuthenticationProperties properties, TokenResponse tokens)
```

Comment:
Well at this point, I was just asking whether that would be better.  Since this seems to be a more general issue, we certainly don't have to do this as part of this change, I'll file an issue for reviewing the names of the Oauth context/methods...